PLEP-0009.rst

```diff
@@ -0,0 +1,97 @@
+=========================================================
+
++-------------------+---------------------------------------------+
+| PLEP              | number                                      |
++===================+=============================================+
+| author(s)         | Nicholas A. Murphy                          |
++-------------------+---------------------------------------------+
+| contact email     | namurphy@cfa.harvard.edu                    |
++-------------------+---------------------------------------------+
+| date created      | *YYYY-MM-DD*                                |
++-------------------+---------------------------------------------+
+| date last revised | *YYYY-MM-DD*                                |
++-------------------+---------------------------------------------+
+| type              | process                                     |
++-------------------+---------------------------------------------+
+| status            | discussion                                  |
++-------------------+---------------------------------------------+
+| DOI               |                                             |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+Abstract
+========
+
+This PLEP describes the nominal release schedule and long-term support
+(LTS) policy for PlasmaPy. 
+
+Detailed Description
+====================
+
+Release schedule
+----------------
+
+Feature releases of PlasmaPy will be scheduled for January, May, and
+September of each year, and maintained with backwards compatible bug
+fixes and documentation improvements until the next feature release.
+
+Python support policy
+---------------------
+
+For consistency with NumPy Enhancement Proposal 29, each feature
+release will be compatible with minor versions of Python released in
+the prior 42 months and minor versions of NumPy released in the prior
+24 months. Because Python minor releases occur each October, the May
+release of PlasmaPy will drop support of the oldest-supported version
+of Python. For example, the May 2025 release of Python will drop
+compatibility with Python 3.10 and require Python 3.11 or newer.
+
+Long-term support policy
+------------------------
+
+If there is sufficient funding for the development and maintenance of
+PlasmaPy, then the first release in even-numbered years will be an LTS
+release that will be maintained with bugfixes and documentation
+improvements for two years.
+
+Implementation
+==============
+
+To implement this PLEP, it will be necessary to implement
+infrastructure for performing backports for LTS releases. For example,
+when a pull request for Astropy is labeled as appropriate for a
+backport, then a GitHub tool will automatically create a pull request
+to the appropriate branches. This infrastructure will reduce the
+effort needed to perform backports and minor releases.
```

Comment:
We should probably delay a decision on an LTS policy until we have better infrastructure in place to do backports. I want to look into how Astropy and other packages have backports set up too.